### PR TITLE
PMM-7586 - HAProxy

### PR DESCRIPTION
**Pull request by IvanaBrecherova**

### CODE REVIEW

**Comment by IvanaBrecherova** [comment]
Tested on: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests-temp/detail/pmm2-ui-tests-temp/416/pipeline

File: tests/pages/pmmInventoryPage.js [hunk 1]
```diff
@@ -81,6 +82,14 @@ module.exports = {
     return serviceIds;
   },
 
+  async getOtherDetails(serviceID, detailsSection, expectedResult) {
```
Comment (puneet0191) [inline tests/pages/pmmInventoryPage.js:85]: what is the reasoning behind such a name of the function? 

Could you improve the function name to clearly represent what it is meant to do, also improve the assert text message, which details section didn't match

Comment (puneet0191) [reply tests/pages/pmmInventoryPage.js:85]: Adding Service name to this failure assert makes more sense too

**Review by puneet0191 — CHANGES_REQUESTED** [review]

File: tests/pages/pmmInventoryPage.js (hunk 1 above)
Comment (IvanaBrecherova) [reply tests/pages/pmmInventoryPage.js:85]: image
The column is named 'Other Details' :) I am going to specify it better :)

**Comment by IvanaBrecherova** [comment]
Build with last changes: https://pmm.cd.percona.com/blue/organizations/jenkins/pmm2-ui-tests/detail/pmm2-ui-tests/8096/pipeline

**Review by puneet0191 — APPROVED** [review]

**Review by yurkovychv — APPROVED** [review]

**Pull request merged**